common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -0,0 +1,103 @@
+import BaseTabs from '../BaseTabs/BaseTabs';
+import { TabType } from '../BaseTabs/BaseTabs';
+import styled from 'styled-components';
+import Space from '../styled/Space';
+import { useContext } from 'react';
+import { AppContext } from '../AppContext/AppContext';
+import Layout12 from '../Layout12/Layout12';
+import { trackEvent } from '@weco/common/utils/ga';
+
+const Tab = styled(Space).attrs({
+  as: 'span',
+  v: { size: 'm', properties: ['padding-top', 'padding-bottom'] },
+  h: { size: 'm', properties: ['padding-left', 'padding-right'] },
+  className: 'flex-inline',
+})`
+  background: ${props => props.theme.color('cream')};
+  border-left: 1px solid ${props => props.theme.color('pumice')};
+  border-top: 1px solid ${props => props.theme.color('pumice')};
+
+  ${props =>
+    props.isLast &&
+    `
+    border-right: 1px solid ${props.theme.color('pumice')};
+  `}
+
+  ${props =>
+    props.isActive &&
+    `
+    background: ${props.theme.color('white')};
+  `}
+
+  ${props =>
+    props.isFocused &&
+    `
+    box-shadow: ${props.isKeyboard ? props.theme.focusBoxShadow : null};
+    position: relative;
+    z-index: 1;
+  `}
+`;
```

Comment:
How does this component know it has `isFocused`?

Reply:
it's an argument passed to the render prop [here](https://github.com/wellcomecollection/wellcomecollection.org/pull/5614/files#diff-57484c78334523af4b4550d1f0423cb8395ef49a3ad6a67d4eb36333abfa8788R55)